requirements.txt

```diff
@@ -17,3 +17,4 @@ PyYAML>=5.4
 s3fs==2021.5.0
 six==1.16.0
 oauth2client==4.1.3
+ruamel.yaml==0.17.20
```

Comment:
I had always used pyyaml so I was looking into the difference..

Some links that I found:
- https://stackoverflow.com/a/36760452/8676953
- https://github.com/yaml/pyyaml/issues/46

So it seems some of the major advantages is YAML 1.2 and being developed more actively.